force-app/main/default/classes/OpenWeatherChecker.cls

```diff
@@ -0,0 +1,51 @@
+/**
+ * File:        OpenWeatherChecker.trigger
+ * Project:     OSFPracticalExercises
+ * Date:        August 19, 2021
+ * Created By:  Can Yavuzkurt
+ * *************************************************************************
+ * Description: Invokes OpenWeather to receive weather condition
+ * *************************************************************************
+ * History:
+ * Date:                Modified By:                         Description:
+ * August 19, 2021      Can Yavuzkurt                        Created
+ * August 20, 2021      Can Yavuzkurt                        Updated endpoint url
+ */
+public with sharing class OpenWeatherChecker {
+    
+    private static String apiKey = 'cd5155cd5e4a5d3751e0e98a4374bcb4';
```

Comment:
as it is constant syntax have to be
private static final String API_KEY = '';

Reply:
Updated